specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json

```diff
@@ -2605,14 +3063,14 @@
             "$ref": "#/definitions/Process"
           }
         },
-        "Ports": {
+        "ports": {
```

Comment:
Could you confirm this is expected and the **exact** case used in the JSON? C# is not case-sensitive, but Python, Node, etc. are case-sensitive. I'm nervous when I see a case change :)

Reply:
yes, it is expected change.